docs/standards/v1/STANDARDS.md

```diff
@@ -0,0 +1,322 @@
+
+## Aggregation
+
+-- TODO
+
+## GeoJSON
+
+APIs will return valid GeoJSON as per [GeoJSON spec](https://geojson.org/)
+
+Proper Field Types should be used for each Feature
+  * Use points for individual Point Objects
+  * Generally MultiPolygons should be avoided
+    - https://gis.stackexchange.com/questions/225368/understanding-difference-between-polygon-and-multipolygon-for-shapefiles-in-qgis
+
+Other related fields/data will be returned as Properties:
+
+  ```json
+  "properties": {
+                 "firstProp": "value0",
+                 "secondProp": {
+                     "this": "that"
+                 }
+             }
+  ```
+
+Lists should be returned via "FeatureCollections":  
+
+  ```json
+    {
+         "type": "FeatureCollection",
+         "features": [{
+             "type": "Feature",
+             "geometry": {
+                 "type": "Point",
+                 "coordinates": [102.0, 0.5]
+             },
+             "properties": {
+                 "firstProp": "value0"
+             }
+         }, {
+             "type": "Feature",
+             "geometry": {
+                 "type": "LineString",
+                 "coordinates": [
+                     [102.0, 0.0],
+                     [103.0, 1.0],
+                     [104.0, 0.0],
+                     [105.0, 1.0]
+                 ]
+             },
+             "properties": {
+                 "firstProp": "value0",
+                 "secondProp": 0.0
+             }
+         }, {
+             "type": "Feature",
+             "geometry": {
+                 "type": "Polygon",
+                 "coordinates": [
+                     [
+                         [100.0, 0.0],
+                         [101.0, 0.0],
+                         [101.0, 1.0],
+                         [100.0, 1.0],
+                         [100.0, 0.0]
+                     ]
+                 ]
+             },
+             "properties": {
+                 "firstProp": "value0",
+                 "secondProp": {
+                     "this": "that"
+                 }
+             }
+         }]
+     }
+  ```
+  * Standard Projection should be [EPSG:4326](https://spatialreference.org/ref/epsg/wgs-84/)
+
+#### Recommended Technologies to Support
+
+* [djangorestframework-gis](https://github.com/djangonauts/django-rest-framework-gis) provides support for DRF for models, views, serializers, returning valid GeoJSON
+* Use with [POSTGIS](https://postgis.net/) database and [Django PostGIS engine](https://docs.djangoproject.com/en/2.2/ref/contrib/gis/tutorial/#configure-settings-py)
+
+## Documentation
+
+Projects should include the following documentation:
+
+* Basic README providing project summary, Hack Oregon intro, quickstart, installation, and contribution guidelines
+* Formatted [docstrings](https://www.django-rest-framework.org/api-guide/schemas/#schemas-as-documentation) on classes and methods
+* Swagger UI frontend
+* Published OpenAPI schema
```

Comment:
> Swagger UI frontend
> Published OpenAPI schema

Would API developers need any additional links or information in this standards document about how to implement these items?

Reply:
Yes we can provided further details, though I do want to get the general guide approved, and moved to master for visibility.

I added this note to spec, which I think we can update in the next 1-2 weeks, once some of the documentation open issues are complete?

https://github.com/hackoregon/2019-backend-cookiecutter-django/commit/e5deb44f4e905c7f76147c1c0d963049d3df8364